lib/vagrant-service-manager/command.rb

```diff
@@ -266,21 +265,17 @@ def print_vagrant_box_version(script_readable = false)
       end
 
       def restart_service(service)
-
-        if SCCLI_SERVICES.include? service
-          # TODO : Handle the case where user wants to pass extra arguments to OpenShift service
-          command = "sudo sccli #{service}"
-        else
-          command = "sudo systemctl restart #{service}"
-        end
-
-        with_target_vms(nil, { single_target: true}) do |machine|
-          machine.communicate.execute(command) do |type, error|
-            if type == :stderr
-              @env.ui.error(error)
-              exit 126
-            end
+        errors = []
+        command = "systemctl restart #{service}"
```

Comment:
kubernetes setup consist of a master and minions. In ADB / CDK, we have a single node setup where master and minion are configured on same node.

For master node, there are following services which are needed to be configured

1. etcd 
2. kube-apiserver 
3. kube-controller-manager 
4. kube-scheduler

and for minion

1. kube-proxy
2. kubelet
3. docker

The sequence of commands to be executed for configuring kubernetes service is written in sccli.
If we do not want to use sccli, we need to execute those commands in plugin.